python/ray/tune/tune.py

```diff
@@ -27,6 +36,16 @@
 }
 
 
+def _print_progress(trial_runner):
+    if IS_NOTEBOOK:
+        from IPython.display import clear_output
+        from IPython.core.display import display, HTML
+        clear_output(wait=True)
+        display(HTML(trial_runner.debug_string(format="html")))
```

Comment:
Probably want to change this somehow so that we don't clear errors/warnings, only the table (maybe theres a way to update a single HTML element?)

Reply:
Let's maybe feature-flag off this printing, but in a separate issue make sure everything in stderr goes into a logfile?

Reply:
can you clarify how you want to flag it off, why not just use the existing `verbose=0`?

Reply:
So I think mainly what would be good is to keep the current default verbosity at 2, and that should be the same;

then, i think for verbosity == 1, we should replace the debug_string with the new one

and then for verbosity == 0, we don't change